Add MNIST in WebDataset format

barci2 opened the pull request:
MNIST-based dataset for development and testing of the Webdataset reader.

barci2 committed:
dataset for the webdataset example and testing

barci2 committed:
Renamed the datasets to a more appropriate one

barci2 committed:
Unpacked 2 more tar datasets for better conformity checking for the webdataset

barci2 committed:
Added the necessary LICENSE and README to the webdataset datasets

barci2 committed:
Removed leftover datasets

jantonguirao commented:
I am wondering if we should include the untarred version as well. It's a lot of files, and we can probably get away with untarring to a temp directory in tests, when necessary.

barci2 commented:
> I am wondering if we should include the untarred version as well. It's a lot of files, and we can probably get away with untarring to a temp directory in tests, when necessary.

Yeah I'll try to update that today

klecki commented:
I think all tared images and jpgs should land in Git LFS. 

~Also, are all files stored separately? Doesn't Webdataset resemble TFRecord and LMDB in the fact that it can pack the data densely?~ 
Misread the extension, I agree we should keep only the .tar here probably.

I would expect something similar to https://github.com/NVIDIA/DALI_extra/pull/34

Also, please put it in webdataset/MNIST/... path, as I expect we might add different webdataset examples in the future.

barci2 committed:
Removed the extracted datasets for them to be unpacked during the actual testing

Signed-off-by: Bartłomiej Cieślar <bcieslar2001@gmail.com>

Removed the extracted datasets for them to be unpacked during the actual testing

Signed-off-by: Bartłomiej Cieślar <bcieslar2001@gmail.com>
